react/PaymentMethod.tsx

```diff
@@ -78,9 +79,9 @@ const PaymentMethod: FunctionComponent<Props & InjectedIntlProps> = ({
         <div className="flex items-center">
           <p className={`${handles.paymentValue} c-muted-1 mv0`}>
             <Price value={payment.value} currency={currency} />
-            {` ${intl.formatMessage(messages.installments, {
+            <span className={`${handles.paymentInstallments}`}>{` ${intl.formatMessage(messages.installments, {
               installments: payment.installments,
-            })}`}
+            })}`}</span>
```

Comment:
Can't this modify current behavior in other stores?

Reply:
This is just a span with a new handle that nobody is using.
I don't see a breaking change here.